plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -78,44 +90,29 @@ export default function MySubscriptions(): JSX.Element {
 
 	return (
 		<div className="woocommerce-marketplace__my-subscriptions">
-			<section>
-				<h2 className="woocommerce-marketplace__my-subscriptions__header">
-					{ __( 'Installed on this store', 'woocommerce' ) }
-				</h2>
-				<p className="woocommerce-marketplace__my-subscriptions__table-description">
-					<span
-						dangerouslySetInnerHTML={ {
-							__html: updateConnectionHTML,
-						} }
-					/>
-					<Tooltip
-						text={
-							<>
-								<h3>
-									{ __(
-										"Still don't see your subscription?",
-										'woocommerce'
-									) }
-								</h3>
-								<p
-									dangerouslySetInnerHTML={ {
-										__html: __(
-											'To see all your subscriptions go to <a href="https://woocommerce.com/my-account/" target="_blank" class="woocommerce-marketplace__my-subscriptions__tooltip-external-link">your account</a> on WooCommerce.com.',
-											'woocommerce'
-										),
-									} }
-								/>
-							</>
-						}
-					>
+			<section className="woocommerce-marketplace__my-subscriptions__installed">
+				<header className="woocommerce-marketplace__my-subscriptions__header">
+					<div>
+						<h2 className="woocommerce-marketplace__my-subscriptions__heading">
+							{ __( 'Installed on this store', 'woocommerce' ) }
+						</h2>
+						<span className="woocommerce-marketplace__my-subscriptions__table-description">
+							{ installedTableDescription }
+						</span>
+					</div>
+					<div>
 						<Button
-							icon={ help }
-							iconSize={ 20 }
-							isSmall={ true }
-							label={ __( 'Help', 'woocommerce' ) }
-						/>
-					</Tooltip>
-				</p>
+							href={ updateConnectionUrl }
+							className="woocommerce-marketplace__refresh-subscriptions"
+						>
+							<img
+								src={ RefreshIcon }
+								alt="Refresh subscriptions"
```

Comment:
Translation missing

Reply:
Thanks! Added in https://github.com/woocommerce/woocommerce/pull/41066/commits/45d85c273054479a9fa3c67c7fc38949d8516146.